### ISSUE TITLE
Focus change to Finder

### ISSUE THREAD

**Issue by bigpal4** [sbmpost/AutoRaise#12]
Great app, I didn't realize this existed. I'm been griping about MacOS focus for years.

I noticed when you have focus on an app like Safari, when you hover over empty desktop, it doesn't change the focus to Finder. I'm not sure if this is the expected behavior, or if I even dislike it. Is this expected, and if so, was that just the behavior you wanted?

**Comment by sbmpost** [comment]
@bigpal4 Good question! I have added some debug output:

    2020-12-11 13:47:55.794 AutoRaise[40667:8134114] Focused window: AutoRaise — AutoRaise — 125×41
    2020-12-11 13:47:55.794 AutoRaise[40667:8134114] Needs raise: 1, window1 id: 0, window2 id: 32480
    2020-12-11 13:47:55.794 AutoRaise[40667:8134114] Raising

This reveals that the tool actually tries to raise the (Finder?) window with id 0, but fails to do so. I recently made some changes to the code which checks if currently focused window is contained within the larger background window and not raise the background window if this is the case. So to answer your question: Previously it was not really intended behaviour but more a side effect. But when my changes are going to the master branch, the behaviour of not raising becomes intended ;-)
